


const a=async()=>{

    try{
        consol.log("shivansh");
    }
    catch(err){
        console.log(err)
    }

}


a()